.circleci/release.sh

```diff
@@ -11,7 +11,7 @@ CIRCLE_TOKEN=${1:-}
 
 ACCOUNT=datadotworld
 PROJECT=aws-sdk-java
-BRANCH=dw-master
+BRANCH=dw-main
```

Comment:
Why not take this opportunity to standardize on `main`?

Reply:
I think it was to differentiate between the main branch of the original repo, but I suppose it doesn't hurt anything